docs/metrics.md

```diff
@@ -0,0 +1,12 @@
+## Datadog Metrics
+Kuberpult uploads mulitple metrics to datadog.
+
+### `cd-service` Metrics
+The cd-service uploads the following metrics to datadog:
+* `env_lock_count` - the count of current environment locks, for a given environment; \[❗ deprecated\]
```

Comment:
this needs to be updated before merging

Reply:
What do you mean by updated? Do you want me to remove the deprecated warning from the docs?